adage/analyze/tests.py

```diff
@@ -366,8 +366,24 @@ def setUp(self):
             **self.test_experiment))
 
         # Create a few annotation types to retrieve with the API
-        self.at_count = 10
-        factory.create(AnnotationType, self.at_count)
+        ann_type = AnnotationType(
+            typename="type name % one",
```

Comment:
So these first 3 test cases check that the ``%, &, #`` characters get encoded appropriately in the url?